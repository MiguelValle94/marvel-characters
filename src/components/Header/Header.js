import React from 'react';
import { Link } from 'react-router-dom';

import './Header.scss';

export default function Header() {
  return (
    <nav className='navbar navbar-light bg-light header'>
      <Link to='/'>
        <img
          src={require('../../assets/marvel_logo.svg')}
          alt='Marvel'
          height='40'
        />
      </Link>
    </nav>
  );
}
