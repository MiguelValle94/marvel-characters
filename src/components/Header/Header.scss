.header {
  justify-content: center;
  margin-bottom: 20px;
  padding: 0;
}
