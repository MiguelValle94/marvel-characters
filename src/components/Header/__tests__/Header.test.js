import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

const setUpShallow = () => {
  const component = shallow(<Header />);
  return component;
};

describe('Header', () => {
  it('should match snapshot', () => {
    const component = setUpShallow();
    expect(component).toMatchSnapshot();
  });
});
