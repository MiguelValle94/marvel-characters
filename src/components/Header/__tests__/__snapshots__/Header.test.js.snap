// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<nav
  className="navbar navbar-light bg-light header"
>
  <Link
    to="/"
  >
    <img
      alt="Marvel"
      height="40"
      src={
        Object {
          "ReactComponent": Object {
            "$$typeof": Symbol(react.forward_ref),
            "render": [Function],
          },
          "__esModule": true,
          "default": "marvel_logo.svg",
        }
      }
    />
  </Link>
</nav>
`;
